docs/walkthrough/README.md

```diff
@@ -16,11 +16,11 @@ tar -zxf sampledataset.v3.tar.gz
 RNAIndel integrates the callset from the built-in caller and that from GAKT (pre-called).<br>
 To do this, input the original BAM file (i.e., sample.bam, not the pre-processed one) and the GATK VCF file:
 ```
-rnaindel PredictIndels -i ./sampledataset/sample.bam \ # do not use sample.gatk.bam
+rnaindel PredictIndels -i sampledateaset/sample.bam \ # do not use sample.gatk.bam
```

Comment:
Just to confirm, is this the right directory name? `sampledataset` vs. `sampledateaset`. I can't download the sample data from the St. Jude research site. It might be worth checking that as well.

Reply:
Yes, it is. The misspelled one is the correct name for the directory. 

The sample name file has the expected name.

`curl -LO http://ftp.stjude.org/pub/software/RNAIndel/sampledataset.v3.tar.gz`

However, the expanded directory has the unexpected name.

I will contact the IT dept to ask for correction.